test/manipulation.test.js

```diff
@@ -984,6 +984,50 @@ describe('manipulation', function() {
         });
       });
 
+      it('should ignore PK if it is set for `instance`' +
+      'in `before save` operation hook', function(done) {
+        function changeIdInBeforeSaveHook() {
```

Comment:
This method is repeated twice (once in each test), is it worth sharing it? If not, then I am proposing to inline it where it's called.